admin/client/App/screens/Item/actions.js

```diff
@@ -23,15 +23,17 @@ export function selectItem (itemId) {
  */
 export function loadItemData () {
 	return (dispatch, getState) => {
+		const thisLoadRef = {};
 		dispatch({
 			type: LOAD_DATA,
+			loadingRef: thisLoadRef,
 		});
 		const state = getState();
 		const list = state.lists.currentList;
 		// Load a specific item with the utils/List.js helper
 		list.loadItem(state.item.id, { drilldown: true }, (err, itemData) => {
 			if (err || !itemData) {
-				console.log('Error loading item data', err);
+				if (getState().loadingRef !== thisLoadRef) return;
```

Comment:
Shouldn't this check move above the `if`? We want to check that the currently loaded item is the item page we're on both if the loading succeeded and if it failed.